DoctrineBundle.php

```diff
@@ -36,9 +36,7 @@ class DoctrineBundle extends Bundle
 {
     private ?Closure $autoloader = null;
 
-    /**
-     * {@inheritDoc}
-     */
+    /** @return void */
```

Comment:
I suggest also tagging the class as `@final since 2.9`. If we were already trating our bundle class as final, we would add actual return types (and there is no valid use case to extend the bundle class IMO)

Reply:
btw, we should also do it for the DI extension and all compiler passes